File: Backend/ProjektZespołówka/Models/Outlet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using static ProjektZespołówka.Models.Helpers;

namespace ProjektZespołówka.Models
{
    public class Outlet
    {
        public int Id { get; set; }
        public Room Room { get; set; }
        public User ServedBy { get; set; }
        public string TechnicalName { get; set; }
        public OutletType Type { get; set; }
        public FileAttachment NearPhoto { get; set; }   
        public FileAttachment FarPhoto { get; set; }
        public OutletStatus Status { get; set; }
        public DateTime InstallationDate { get; set; }
    }
}

File: mobile/composeApp/src/commonMain/kotlin/com/rackncode/cloudwire/domain/model/Outlet.kt
package com.rackncode.cloudwire.domain.model

import com.rackncode.cloudwire.domain.helpers.OutletStatus
import com.rackncode.cloudwire.domain.helpers.OutletType
import kotlinx.datetime.LocalDateTime

data class Outlet(
    val id: String,
    val roomId: String,
    val servicedById: String,
    val technicalName: String,
    val type: OutletType,
    val closeUpPhotoId: String,
    val farPhotoId: String,
    val status: OutletStatus,
    val installationDate: LocalDateTime
)
